### PR TITLE
Jmacd/duration 64bit

### PR DESCRIPTION
**Pull request by jmacd**

### CODE REVIEW

**Comment by jmacd** [comment]
(Spoons says you're against 64bit durations)

File: collector.proto [hunk 1]
```diff
@@ -40,15 +40,15 @@ message Span {
     string operation_name = 2;
     repeated Reference references = 3;
     google.protobuf.Timestamp start_timestamp = 4;
-    uint32 duration_micros = 5;
+    uint64 duration_micros = 5;
```
Comment (bhs) [inline collector.proto:43]: I'll live with it :)

I _am_ against 96-bit durations, though (i.e., `google.protobuf.Duration`).

**Comment by bhs** [comment]
:ship: 😿

File: collector.proto (hunk 1 above)
Comment (jmacd) [reply collector.proto:43]: 93.897352854

**Pull request merged**